File: Java/Examen2Java/Fecha.java
package Examen2Java;

public class Fecha {
    private int dia;
    private int mes;
    private int año;

    public Fecha(int dia, int mes, int año){
        this.dia = dia;
        this.mes = mes;
        this.año = año;
    }

    public Fecha(){}

    public void setDia(int dia) {
        this.dia = dia;
    }

    public int getDia() {
        return dia;
    }

    public void setMes(int mes) {
        this.mes = mes;
    }

    public int getMes() {
        return mes;
    }

    public void setAño(int año) {
        this.año = año;
    }

    public int getAño() {
        return año;
    }
}

File: Python/Ejercicios02/Fecha.py
class Fecha:
    def __init__(self, dia = 0, mes = 0, año = 0):
        self.__dia = dia
        self.__mes = mes
        self.__año = año
        
    def setDia(self, dia):
        self.__dia = dia

    def getDia(self):
        return self.__dia
    
    def setMes(self, mes):
        self.__mes = mes

    def getMes(self):
        return self.__mes
    
    def setAño(self, año):
        self.__año = año
    
    def getAño(self):
        return self.__año